tests/cnf/core/network/policy/tests/multinetpolicy_bondoversriov.go

```diff
@@ -0,0 +1,729 @@
+		Expect(err).ToNot(HaveOccurred(), "ingress rule configuration not generated")
+
+		_, err = networkpolicy.NewMultiNetworkPolicyBuilder(APIClient, "ingress-egress", "policy-ns1").
+			WithNetwork("policy-ns1/bond,policy-ns2/bond").
+			WithPodSelector(metav1.LabelSelector{MatchLabels: map[string]string{"app": "pod1"}}).
+			WithPolicyType(multinetpolicyapiv1.PolicyTypeIngress).
+			WithPolicyType(multinetpolicyapiv1.PolicyTypeEgress).
+			WithIngressRule(*testIngressRule).
+			WithEgressRule(*testEgressRule).
+			Create()
+		Expect(err).NotTo(HaveOccurred(), "failed to create multi network policy")
+
+		By("Check egress traffic from pod1 to other 4 pods. Only Pod5 ports should be accessible over IPv6")
+		defineNmapValidPaths(
+			[]*pod.Builder{testPod1}, []*pod.Builder{testPod2, testPod3, testPod4, testPod5},
+			[]int64{0o00, 0o00, 0o00, 0o07}, testData)
+
+		By("Check ingress traffic to pod1 from other 4 pods. " +
+			"Pod2 can access tcp ports 5001 & 5002 of pod1 over IPv4. " +
+			"Pod4 can access tcp ports 5001 & 5002 of pod1 over both IPv4 & IPv6")
+		defineNmapValidPaths(
+			[]*pod.Builder{testPod2, testPod3, testPod4, testPod5}, []*pod.Builder{testPod1},
+			[]int64{0o60, 0o00, 0o66, 0o00}, testData)
+	})
+})
+
+func defineAndCreateSriovNetwork(netName, resName, targetNs string) {
+	_, err := sriov.NewNetworkBuilder(
+		APIClient, netName, NetConfig.SriovOperatorNamespace, targetNs, resName).
+		WithLogLevel(netparam.LogLevelDebug).
+		Create()
+	Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("Failed to create Sriov Network %s", netName))
+}
+
+func defineAndCreateBondNAD(nadName, nsName string) {
+	config, err := nad.NewMasterBondPlugin("bond", "active-backup").
+		WithFailOverMac(1).
+		WithLinksInContainer(true).
+		WithMiimon(100).
+		WithLinks([]nad.Link{{Name: "net1"}, {Name: "net2"}}).
+		WithCapabilities(&nad.Capability{IPs: true}).
+		WithIPAM(&nad.IPAM{
+			Type:   "static",
+			Routes: []nad.Routes{{Dst: "192.168.0.0/16"}, {Dst: "2001::0/62"}},
+		}).GetMasterPluginConfig()
+	Expect(err).ToNot(HaveOccurred(), "Failed to get master bond plugin config")
+
+	_, err = nad.NewBuilder(APIClient, nadName, nsName).WithMasterPlugin(config).Create()
+	Expect(err).ToNot(HaveOccurred(), "Failed to create net-attach-def")
+}
+
+func defineAndCreatePodWithBondIf(podName, nsName, net1Name, net2Name string, testData podsData) *pod.Builder {
+	var rootUser int64
+
+	securityContext := corev1.SecurityContext{
+		RunAsUser: &rootUser,
+		Capabilities: &corev1.Capabilities{
+			Add: []corev1.Capability{"IPC_LOCK", "SYS_RESOURCE", "NET_RAW", "NET_ADMIN"},
+		},
+	}
+
+	netAnnotation := []*types.NetworkSelectionElement{
+		{
+			Name:             net1Name,
+			InterfaceRequest: "net1",
+		},
+		{
+			Name:             net2Name,
+			InterfaceRequest: "net2",
+		},
+		{
+			Name:             "bond",
+			InterfaceRequest: "bond1",
+			IPRequest:        append(append([]string{}, testData[podName].IPv4), testData[podName].IPv6),
```

Comment:
nit: inline arrays can be created with:

```suggestion
			IPRequest:        []string{testData[podName].IPv4, testData[podName].IPv6},
```

Reply:
done